lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -591,6 +591,355 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
+        Value lfDiv7HAdd1 = b.create<Torch::AtenAddScalarOp>(
+            freqType, lfDiv7H, oneConst, /*alpha =*/oneConst);
+        Value lfDiv7HAdd1Log10 =
+            b.create<Torch::AtenLog10Op>(freqType, lfDiv7HAdd1);
+        Value lfMel = b.create<Torch::AtenMulScalarOp>(
+            freqType, lfDiv7HAdd1Log10, twoFiveNineFiveConst);
+
+        Value hfDiv7Hfloat = b.create<Torch::AtenDivFloatOp>(
+            upperEdgeHzItem,
+            sevenHConst); // Since all others need tensor to operate
+        Value hfDiv7H =
+            b.create<Torch::PrimNumToTensorScalarOp>(freqType, hfDiv7Hfloat);
+        Value hfDiv7HAdd1 = b.create<Torch::AtenAddScalarOp>(
+            freqType, hfDiv7H, oneConst, /*alpha =*/oneConst);
+        Value hfDiv7HAdd1Log10 =
+            b.create<Torch::AtenLog10Op>(freqType, hfDiv7HAdd1);
+        Value hfMel = b.create<Torch::AtenMulScalarOp>(
+            freqType, hfDiv7HAdd1Log10, twoFiveNineFiveConst);
+
+        Value hfSubLf = b.create<Torch::AtenSubTensorOp>(
+            hfMel.getType(), hfMel, lfMel, /*alpha=*/oneConst);
+        Value melStep = b.create<Torch::AtenDivScalarOp>(
+            hfSubLf.getType(), hfSubLf, numMelBinsItem);
+
+        Value freqBinsMulMelStep = b.create<Torch::AtenMulTensorOp>(
+            freqBinsFltType, freqBinsInit, melStep);
+        Value freqBinsScaled = b.create<Torch::AtenAddTensorOp>(
+            freqBinsFltType, freqBinsMulMelStep, lfMel, /*alpha=*/oneConst);
+
+        // Mel to Hz conv
+
+        Value fbDiv = b.create<Torch::AtenDivScalarOp>(
+            freqBinsFltType, freqBinsScaled, twoFiveNineFiveConst);
+        Value fbClone = b.create<Torch::AtenCloneOp>(
+            freqBinsFltType, freqBinsScaled, /*memory_format=*/noneConst);
+        Value tenTensor = b.create<Torch::AtenFillScalarOp>(freqBinsFltType,
+                                                            fbClone, tenConst);
+        Value fbPow = b.create<Torch::AtenPowTensorTensorOp>(freqBinsFltType,
+                                                             tenTensor, fbDiv);
+        Value fbPowSubOne = b.create<Torch::AtenSubScalarOp>(
+            freqBinsFltType, fbPow, oneConst, /*alpha=*/oneConst);
+        Value freqBinsHz = b.create<Torch::AtenMulScalarOp>(
+            freqBinsFltType, fbPowSubOne, sevenHConst);
+
+        // Normalize freqBinsHz
+        Value dftLenPlusOne = b.create<Torch::AtenAddScalarOp>(
+            dftLenType, operands[1], oneConst, /*alpha=*/oneConst);
+        Value dftLenPlusOneItem =
+            getItemOp<Torch::IntType>(binder, rewriter, dftLenPlusOne);
+        Value fbMulDft = b.create<Torch::AtenMulScalarOp>(
+            freqBinsFltType, freqBinsHz, dftLenPlusOneItem);
+        Value freqBinsNormalized = b.create<Torch::AtenDivScalarOp>(
+            freqBinsFltType, fbMulDft, sampleRateItem);
+
+        // cast to int32
+        Value int32TypeConst =
+            b.create<Torch::ConstantIntOp>(rewriter.getI64IntegerAttr(3));
```

Comment:
`int32TypeConst`? Doesn't match with the actual value.

Reply:
Value `3` is the enum value for Torch dtype of int32. Needed for Dtype conversion.

Reply:
Change `int32TypeConst` -> `int32DtypeConst`